docs/modules/ROOT/pages/contributing/local-development.adoc

```diff
@@ -65,4 +65,20 @@ Camel K integrations are based on https://github.com/apache/camel-k-runtime[Came
 ./kamel install --skip-operator-setup -n operator-test --registry localhost:5000 --force --runtime-version 1.7.0-SNAPSHOT
 ----
 
-With the above instructions, the operator will pick up and use the snapshot version you have released locally.
\ No newline at end of file
+With the above instructions, the operator will pick up and use the snapshot version you have released locally.
+
+==== Using a Docker registry
+
+Alternatively, if no local registry is available, you can use a docker registry. First, create a secret with your registry username and password:
+
+```
+kubectl -n default create secret docker-registry external-registry-secret --docker-username my-user --docker-password "password"
+```
+
+And then you can run the installation of Camel K operator:
+
+```
+./kamel install --skip-operator-setup -n operator-test --force --runtime-version 1.7.0-SNAPSHOT --registry docker.io --organization my-org-or-username --registry-secret external-registry-secret 
```

Comment:
I don't think we should use 1.7.0-SNAPSHOT as runtime version as default in the docs.

Reply:
I am using the same version number that is in the same documentation page. Let me replace it with a variable.